### PR TITLE
Add definition for 'ggplot2' (English)

### PR DESCRIPTION
**Pull request by corybrunson**
Fill in each of the sections (using NA for those that are not applicable).

## Author: 

- Jason Cory Brunson

## Language: 

- English

## Terms defined:

- ggplot2

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -939,7 +939,7 @@
     term: "Umkhethe"
     def: >
       Ubalo-manani lunomkhethe ukuba lwenziwe ngendlela echanekileyo okanye ngokungaguquguqukiyo ngokwahlukileyo kwiparameter emele ukuba iqikelele.
-  
+
```
Comment (zkamvar) [inline glossary.yml:942]:
```suggestion
```

**Review by zkamvar — CHANGES_REQUESTED** [review]
This looks good to me. Thank you!

Note that there might be some merge conflicts due to the editor deletions of trailing whitespace characters, but I should be able to address that before I merge.

**Comment by corybrunson** [comment]
Oh my goodness, i'm sorry about that—i should have checked the `diff` before creating the PR. I'm glad to make a correction commit if it would make things more convenient on your end!

**Comment by zkamvar** [comment]
> Oh my goodness, i'm sorry about that—i should have checked the `diff` before creating the PR. I'm glad to make a correction commit if it would make things more convenient on your end!

No worries! Part of the purpose of this repo is for demonstration and practise of git skills in a low-stakes environment, so these situations are perfectly normal and expected :smiley: The only thing I will ask is to click the "commit suggestion" button on my suggestion: https://github.com/carpentries/glosario/pull/357#discussion_r685325148

**Comment by corybrunson** [comment]
Aha, thank you @zkamvar – that feature was new to me and i appreciate the lesson.

**Review by zkamvar — APPROVED** [review]

**Pull request merged**